doc/getting-started/gsg_quick_setup.sh

```diff
@@ -0,0 +1,277 @@
+echo -e "\t-u to upgrade UOS."
+echo -e "\t-s, skip downloading uos; if enabled, you have to download uos img firstly before upgrading, default is off"
+exit 1
+}
+
+# switch for download uos function
+skip_download_uos=0
+# turn on if you need to setup uos
+upgrade_uos=0
+
+# acrn.conf path
+acrn_conf_path=/usr/share/acrn/samples/nuc/acrn.conf
+# acrn.efi path
+acrn_efi_path=/usr/lib/acrn/acrn.efi
+
+function upgrade_sos()
+{
+# setup mirror and proxy url while specified with m and p options
+if [[ -n $mirror ]]; then
+    echo "Setting swupd mirror to: $mirror"
+    swupd mirror -s $mirror
+fi
+if [[ -n $proxy ]]; then
+    echo "Setting proxy to: $proxy"
+    export https_proxy=$proxy
+fi
+
+# get partitions
+type_efi=EFI\ System
+partition_number=`fdisk -l | grep "$type_efi" | awk -F' ' '{print $1}' | wc -l`
+if [ $partition_number != "1" ]; then
+    echo "Multiple partitions detected."
+    c=0
+    for i in `fdisk -l | grep "$type_efi" | awk -F' ' '{print $1}'`; do
+        if [[ ! -z $efi_partition ]]; then break; fi
+        c=$((c+1))
+        echo "$c. $i"
+        while true; do
+            read -p "Do you wish to setup on this partition ? (Y/N)" yn
+            case $yn in
+                [Yy]* ) efi_partition=$i; break;;
+                [Nn]* ) break;;
+                * ) echo "Please input yes or no.";;
+            esac
+        done
+    done
+else
+    efi_partition=`fdisk -l | grep "$type_efi" | awk -F' ' '{print $1}'`
+fi
+
+if [[ -z $efi_partition ]]; then echo "You didn't choose a partition, please try it again."; exit 1; fi
+efi_mount_point=`findmnt $efi_partition -n | cut -d' ' -f1`
+root_partition=`echo $efi_partition | sed 's/[0-9]$/3/g'`
+partition=`echo $efi_partition | sed 's/[0-9]$//g'`
+# unmount efi partition if it's already mounted
+if [[ -n $efi_mount_point ]]; then
+    echo "Unmount EFI partition: $efi_mount_point..."
+    umount $efi_mount_point
+fi
+
+echo "Disable auto update..."
+swupd autoupdate --disable
+
+# Compare with current clear linux and skip upgrade sos if get the same version.
+source /etc/os-release
+if [[ $VERSION_ID -eq $target_version ]]; then
+    echo "Specify with the same clear linux version, continue to setup sos..."
+else
+    echo "Upgrade clear linux version to : $target_version ..."
+    swupd verify --fix --picky -m $target_version
+fi
+
+# Do the setups if previous process succeed.
+if [[ $? -eq 0 ]]; then
+    echo "Add service-os kernel-iot-lts2018 bundle"
+    swupd bundle-add service-os kernel-iot-lts2018
+
+    mount $efi_partition /mnt
+    echo "Add /mnt/EFI/acrn folder"
+    mkdir -p /mnt/EFI/acrn
+    echo "Copy $acrn_conf_path /mnt/loader/entries/"
+    if [[ ! -f $acrn_conf_path ]]; then
+        echo "Missing acrn.conf file in folder: $acrn_conf_path"
+        umount /mnt && sync
+        exit 1
+    fi
+    cp -r $acrn_conf_path /mnt/loader/entries/
+    if [[ $? -ne 0 ]]; then echo "You didn't choose an right EFI partition." && exit 1; fi
```

Comment:
The user can't choose the EFI partition, this is figured out by the script. What's the real error here?

Reply:
Update message in next release, it's checking whether line: `cp -r $acrn_conf_path /mnt/loader/entries/` is running successfully.